run_examples

```diff
@@ -0,0 +1,11 @@
+echo "[info] Running sum of array"
```

Comment:
Instead of this script, you could add a read-me to the examples directory.

Reply:
I can add the README file. But removing the script means the user has to manually execute each example. Wouldn't that be inconvenient?

Reply:
I believe that the purpose of the examples would be to check out the code, change a little and then execute it. I don't see the reason to execute all the examples at once like they were tests or benchmarks.

Reply:
Fair enough. I'll remove the script.